core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -426,7 +426,9 @@ object ConfigCommand extends Config {
       dynamicConfigSource
 
     val configResource = new ConfigResource(configResourceType, entityName)
-    val configs = adminClient.describeConfigs(Collections.singleton(configResource)).all.get(30, TimeUnit.SECONDS)
+    val describeOptions = new DescribeConfigsOptions().includeSynonyms(includeSynonyms)
```

Comment:
I was going to ask about unused linter warnings to catch bugs like this, but turns out we have a problem in our build. Filed https://issues.apache.org/jira/browse/KAFKA-9665